packages/template-retail-react-app/app/commerce-api/auth.js

```diff
@@ -26,14 +26,14 @@ import Cookies from 'js-cookie'
  * @typedef {Object} Customer
  */
 
-const usidStorageKey = 'usid'
-const cidStorageKey = 'cid'
-const encUserIdStorageKey = 'enc-user-id'
-const tokenStorageKey = 'token'
-const refreshTokenRegisteredStorageKey = 'cc-nx'
-const refreshTokenGuestStorageKey = 'cc-nx-g'
-const oidStorageKey = 'oid'
-const dwSessionIdKey = 'dwsid'
+const usidStorageKey = '_usid'
```

Comment:
Rename the storage name so it won't get conflict with the ones from `commerce-sdk-react`